frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -275,5 +294,82 @@ function CollectionSectionHeading({
     </SidebarHeadingWrapper>
   );
 }
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default MainNavbarView;
+
+function SidebarOnboardingSection({ initialState }: { initialState: boolean }) {
+  const applicationName = useSelector(getApplicationName);
+
+  return (
+    <Box
+      m={0}
+      bottom={0}
+      pos="fixed"
+      w={NAV_SIDEBAR_WIDTH}
+      bg="bg-white"
+      className={cx({ [CS.borderTop]: !initialState })}
+    >
+      <Box px="md" py="md">
+        {/*eslint-disable-next-line no-unconditional-metabase-links-render -- This link is only temporary. It will be replaced with an internal link to a page. */}
+        <ExternalLink href={getLearnUrl()} className={CS.noDecoration}>
```

Comment:
MS2 will be all about the new onboarding checklist page (similar to the one we already have in the admin settings). Once that page is available, we'll switch from the "learn" content to routing to this new page.